functions/cors-go/README.md

```diff
@@ -25,3 +35,21 @@ curl -i -X OPTIONS <function URL>
 # Get request
 curl -i -X GET <function URL>
 ```
+
+The result should be similar to:
+
+```console
+HTTP/1.1 200 OK
```

Comment:
I got `HTTP/2 200` with curl, can you replace this for consistency with other readme files ?

Reply:
Solved.